boltdesignsystem/bolt
# Feature/code formatting

## Jira

N/A

## Summary

Enables several CSS code formatting rules

## Details

For each line you remove from packages/configs/stylelint-config/index.js, you can run `yarn lint:scss` to see if there were any violations in the existing codebase.  If there are, `yarn lint:scss --fix` will fix them.  This is an example of doing that for several of the rules.

## How to test

- Confirm that `yarn lint:scss` passes (this will be run as part of Travis build)
- Confirm no regressions or undesirable formatting changes
